docs/source/requests.md

```diff
@@ -195,6 +255,22 @@ of a transaction in the Ledger (see [transactions](transactions.md)).
         - `reqId` (integer): 
             Unique ID number of the request with transaction.
   
+        - `digest` (SHA256 hex digest string):
+            SHA256 hash hex digest of all fields in the initial requests (including signatures) 
+            
+        - `payloadDigest` (SHA256 hex digest string):
+            SHA256 hash hex digest of the payload fields in the initial requests, that is all fields excluding signatures and plugins-added ones
+            
+        - `taaAcceptance` (dict, optional):
+            If transaction author agreement is set/enabled, then every transaction (write request) from Domain and plugins-added ledgers must include acceptance of the latest transaction author agreement.
+            
+                - `taaDigest` (SHA256 hex digest string): SHA256 hex digest of the latest Transaction Author Agreement on the ledger
```

Comment:
I think it should be noted either here or somewhere else in this document that digest is calculated from concatenation of TAA version and text.